test/integration/user_can_create_account_test.rb

```diff
@@ -0,0 +1,64 @@
+require "test_helper"
+
+class UserCanCreateAccountTest < ActionDispatch::IntegrationTest
+  test "user can create account and sees profile" do
+    items = create_list(:item, 2)
+
+    visit items_path
+
+    first(:button, "Add to Cart").click
+    assert page.has_content?("You added #{items.first.title} to your cart.")
```

Comment:
I don't think we need to make this assertion because we're already doing it in another test. Maybe we can break out a method to add items to the cart to help cut down on this test length.